CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ add_subdirectory(doc/examples/planning_scene)
 add_subdirectory(doc/examples/planning_scene_ros_api)
 add_subdirectory(doc/examples/realtime_servo)
 add_subdirectory(doc/examples/robot_model_and_robot_state)
+add_subdirectory(doc/examples/trajectory_cache)
```

Comment:
Thanks for adding a tutorial! I know that this is still WIP, but just figured I'd mention:

Please add your tutorial to the how to guides! The examples are for older tutorials, including some that are unmigrated from ROS 1. The page has this big ol' section at the top:
![image](https://github.com/user-attachments/assets/dd1baf41-7ab8-4a32-ac2c-c0c138438487)

I wouldn't want your tutorial to get lost in there.

Reply:
> Thanks for adding a tutorial! I know that this is still WIP, but just figured I'd mention:
> 
> Please add your tutorial to the how to guides! The examples are for older tutorials, including some that are unmigrated from ROS 1. The page has this big ol' section at the top: ![image](https://private-user-images.githubusercontent.com/10466537/351940037-dd1baf41-7ab8-4a32-ac2c-c0c138438487.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4ODEwNDAsIm5iZiI6MTcyMTg4MDc0MCwicGF0aCI6Ii8xMDQ2NjUzNy8zNTE5NDAwMzctZGQxYmFmNDEtN2FiOC00YTMyLWFjMmMtYzBjMTM4NDM4NDg3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI1VDA0MTIyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUyMzE2MzU5YThlNDJlM2I3MDI4YzAwMmUwZGUxYjA4OGVjNjdiMzlmOWNjZjQyMjRlODI3OGYyNWU5YWMyNjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.IVj9dNt5HtmUMgvnnLDV8HhD8gkTubk5fDZF93T_tHQ)
> 
> I wouldn't want your tutorial to get lost in there.

Oh! What if I don't have the time to write a full tutorial, but just wanted to add a usage example for a PR that is in flight? Where is the best place for that?

The PR in question is:
- https://github.com/moveit/moveit2/pull/2838

Reply:
Sounds like a how-to is the perfect place! How to guides are meant for showing off examples for using a specific concept, while the full tutorials are more guided content for a new user. If you want to know more, you can read more about the whole how to vs. tutorial [here](https://moveit.picknik.ai/main/doc/how_to_guides/how_to_guide.html). 

I don't recommend any new work is put in the examples, as we're trying to move away from that.

Appreciate the work and look forward to seeing your talk at ROSCon!